Frontend/src/Screens/GlobalComponents/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+// Import useLocation hook
+import { Link, useLocation } from 'react-router-dom';
+import './NavBar.css';
+
+interface TUserLocation{
+    from:{pathname: string};
+    prevState: null;
+    location:any;
+  }
+const NavBar:React.FC<TUserLocation>=()=> {
+    const location = useLocation(); // once ready it returns the 'window.location' object
+    const [url, setUrl] = useState<string | null>(null);
+    useEffect(() => {
+        setUrl(location.pathname);
+    }, [location.pathname]);
+
+    return (
+        <nav className='navbar-container'>
```

Comment:
For class names, the naming convention should be as such:
ComponentName__DivPurpose__ChildDivPurpose

in this case the naming convention would be Navbar__NavbarContainer.

Please change the class naming convention

If you have any more questions about naming conventions, don't hesitate to ask me.